### ISSUE TITLE
This is adding export statement on CSS files too. 

### ISSUE THREAD

**Issue by gs-akhan** [purtuga/esm-webpack-plugin#6]
Hi, Is there a way to skip this for CSS files ?

Thanks

**Comment by gs-akhan** [comment]
If I add a check to skip .css files. Would that be a fix ?
https://github.com/purtuga/esm-webpack-plugin/blob/master/esm-webpack-plugin.js#L59

Thanks

**Comment by purtuga** [comment]
Short term fix for you is to add that ‘if’ condition in the line you highlighted. I’ll try to post code here with specific as soon as I get a chance to recreate it.

From this plugin standpoint, the change will likely introduce a ‘filter’ option (or ‘skip’) that will allow you define a function to handle what should be skipped. Need to think a little more about that.

Paul

-- sent from mobile

**Comment by gs-akhan** [comment]
should I send a PR ? With this minor if
Thanks

**Comment by purtuga** [comment]
No, don’t push that fix - it’s not robust enough and I rather not publish it.  CSS is not the only other type of file that webpack loaders support, so the fix needs to be a little more flexible.

I will try to work on this today.

Paul

-- sent from mobile

**Comment by gs-akhan** [comment]
sounds good, thank you very much

**Comment by purtuga** [comment]
Can you post a code sample that shows how you are bundling `.css`? Would like to see both your webpack `modules.rule` as well as a code sample with your `import .... somefile.css` in it. 

I just tried this using `style-loader` and `file-loader` plugins, and did not experience the issue. CSS file had no ESM statements in it.

**Comment by gs-akhan** [comment]
sure i can post it in 1.5 hours .

**Comment by gs-akhan** [comment]
These are rules and plugins

```
[
				{
					test: /\.tsx?$/,
					loader: 'awesome-typescript-loader',
					options: {
						sourceMap: currentBundleProfile.sourceMap,
						silent: currentBundleProfile.silent,
						useTranspileModule: currentBundleProfile.transpileOnly,
						transpileOnly: currentBundleProfile.transpileOnly,
						forceIsolatedModules: true,
						useCache: currentBundleProfile.useCache
					}
				}, // use babel-loader along with this to support unused exports
				{
					test: /\.(jpe?g|png|gif)$/i,
					loader: "url-loader",
					options: {
						limit: 8000,
						name: "images/[hash].[ext]"
					}
				},
				{
					test: /(\.scss)|(\.css)/,
					exclude: [/\.__lazy__.scss/], //Exclude any scss file ends with __lazy__, example: emailstyles.__lazy__.scss
					use: [
							MiniCssExtractPlugin.loader,
							{
								loader: 'css-loader',
								options: {
									minimize: currentBundleProfile.minimize,
									sourceMap: currentBundleProfile.sourceMap
								}
							}, {
								loader: "sass-loader",
								options: {
									sourceMap: currentBundleProfile.sourceMap
								}
							}
						]
				},
				{
					test: /\.svg$/, // for resolving svg
					use: [
						'file-loader?name=images/[hash].[ext]'
					]
				},
				{
					test: /\.(woff|woff2|eot|ttf|otf)$/, // for resolving fonts
					use: [
						'file-loader?name=fonts/[hash].[ext]'
					]
				},
				{test: /\.xml$/, loader: 'xml-loader'},
				{
					test: /\.worker\.(js|ts)$/,
					use: {
						loader: 'worker-loader',
						options: {inline: true}
					}
				},
				{
					test: /\.(mp3)$/,
					use: [
						{
							loader: 'file-loader',
							options: {
								name: '[path][name].[ext]'
							}
						}
					]
				}
			]
```

Plugins : 
```
[
		new CleanWebpackPlugin([moduleOutFolder], {
			verbose: !currentBundleProfile.silent
		}),
		new webpack.ProvidePlugin({
			"$": "jquery",
			"j$": "jquery",
			jQuery: "jquery",
			"window.jQuery": "jquery",
			_: "underscore",
			"Backbone": "backbone"
		}),
		new CheckerPlugin(),
		new MiniCssExtractPlugin({
			filename: "styles/[name].css"
		}),
		new webpack.DefinePlugin({'process.env': {LATER_COV: false}}), //For Later js,
		new webpack.IgnorePlugin(/(jsdom|xmldom)/)
	]
```

**Comment by purtuga** [comment]
Thanks. I see you are using a plugin named ‘MiniCssExtractPlugin’. I will look into setting up a quick prototype to recreate the condition using that plugin.

Paul

-- sent from mobile

**Comment by gs-akhan** [comment]
Thank you very much really appreciate all your efforts .. :))

**Comment by purtuga** [comment]
I just pushed a change that by default only adds the ESM exports to file whose extension ends with `.js` or `.mjs`. So without any changes to your webpack configuration, this should be fine now.  The plugin now also support an `exclude` option - a callback function (see [REAME](https://github.com/purtuga/esm-webpack-plugin/blob/master/README.md#options) file for more).

Could you test this before I push to NPM?

Just use the following command to install it directly from github:

```
npm i purtuga/esm-webpack-plugin
```

Then run your build again and let me know if all looks good. I will then release it to NPM.

**Comment by gs-akhan** [comment]
Validated. It is working perfectly. Please publish it. 

Thanks

**Comment by purtuga** [comment]
Awesome. Thanks for validating. I have just published version 1.1.0.
Thanks for reporting this.
